_posts/2019-3-29-Copr-new-rss-feed.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Copr has a new RSS feed
+author: Jan Buchmaier
+layout: post
+---
+
+Now you can keep track off of every new Copr thanks to our new feed.
```

Comment:
> track off of

I think, that both "track off of" and "track of" are correct, but I find "track of" much more appealing.

> every new Copr

AFAIK we have decided to prefer "project" over "copr (in the meaning of the project)". I would go with "every new project" or "every new Copr project".